### PR TITLE
Adds IQE_RP_ARGS reportportal and IQE_IBUTSU_SOURCE args to CJIs

### PR DESCRIPTION
**Pull request by san7ket**
There are two variables those were added to iqecore to integrate with report-portal and also specify ibutsu-source for the test run

### CODE REVIEW

**Comment by san7ket** [comment]
cc @bsquizz

File: apis/cloud.redhat.com/v1alpha1/clowdjobinvocation_types.go [hunk 1]
```diff
@@ -89,6 +89,12 @@ type IqeJobSpec struct {
 
 	// sets value passed to IQE 'IQE_PARALLEL_WORKER_COUNT' arg
 	ParallelWorkerCount string `json:"parallelWorkerCount,omitempty"`
+
+	// sets value passed to IQE 'IQE_RP_ARGS' report portal args
+	RpArgs string `json:"RpArgs,omitempty"`
```
Comment (bsquizz) [inline apis/cloud.redhat.com/v1alpha1/clowdjobinvocation_types.go:94]: The JSON property name should use `camelCase`

```suggestion
	RpArgs string `json:"rpArgs,omitempty"`
```

File: apis/cloud.redhat.com/v1alpha1/clowdjobinvocation_types.go [hunk 2]
```diff
@@ -89,6 +89,12 @@ type IqeJobSpec struct {
...
+
+	// sets value passed to IQE 'IQE_IBUTSU_SOURCE' args
+	IbutsuSource string `json:"IbutsuSource,omitempty"`
```
Comment (bsquizz) [inline apis/cloud.redhat.com/v1alpha1/clowdjobinvocation_types.go:97]: Same here

```suggestion
	IbutsuSource string `json:"ibutsuSource,omitempty"`
```

File: config/crd/bases/cloud.redhat.com_clowdjobinvocations.yaml [hunk 3]
```diff
@@ -57,6 +57,14 @@ spec:
                     description: Iqe is the job spec to override defaults from the
                       ClowdApp's definition of the job
                     properties:
+                      IbutsuSource:
```
Comment (bsquizz) [inline config/crd/bases/cloud.redhat.com_clowdjobinvocations.yaml:60]: When you update the above properties, just make sure to `make pre-push` and the property names here should get updated.

File: config/crd/bases/cloud.redhat.com_clowdjobinvocations.yaml [hunk 4]
```diff
@@ -57,6 +57,14 @@ spec:
...
+                        description: sets value passed to IQE 'IQE_IBUTSU_SOURCE'
+                          args
+                        type: string
+                      RpArgs:
```
Comment (bsquizz) [inline config/crd/bases/cloud.redhat.com_clowdjobinvocations.yaml:64]: `make pre-push` will probably take care of this

File: tests/kuttl/test-iqe-jobs-selenium/01-pods.yaml [hunk 5]
```diff
@@ -91,3 +91,5 @@ spec:
       filter: "test_plugin_accessible"
       parallelEnabled: "true"
       parallelWorkerCount: "4"
+      RpArgs: "true"
```
Comment (bsquizz) [inline tests/kuttl/test-iqe-jobs-selenium/01-pods.yaml:94]: Update to camelCase here

File: tests/kuttl/test-iqe-jobs-vault/01-pods.yaml [hunk 6]
```diff
@@ -95,3 +95,5 @@ spec:
       filter: "test_plugin_accessible"
       parallelEnabled: "true"
       parallelWorkerCount: "4"
+      RpArgs: "true"
```
Comment (bsquizz) [inline tests/kuttl/test-iqe-jobs-vault/01-pods.yaml:98]: Here as well

File: tests/kuttl/test-iqe-jobs/01-pods.yaml [hunk 7]
```diff
@@ -89,3 +89,5 @@ spec:
       - "high"
       parallelEnabled: "true"
       parallelWorkerCount: "4"
+      RpArgs: "true"
```
Comment (bsquizz) [inline tests/kuttl/test-iqe-jobs/01-pods.yaml:92]: And here :)

File: tests/kuttl/test-iqe-jobs/03-pods-with-view.yaml [hunk 8]
```diff
@@ -79,3 +79,5 @@ spec:
       filter: "test_plugin_accessible"
       parallelEnabled: "true"
       parallelWorkerCount: "4"
+      RpArgs: "true"
```
Comment (bsquizz) [inline tests/kuttl/test-iqe-jobs/03-pods-with-view.yaml:82]: Here as well

**Review by bsquizz — CHANGES_REQUESTED** [review]
Looks good, just need to follow the json property naming convention

**Review by bsquizz — APPROVED** [review]

**Pull request merged**